### PR TITLE
V8 probe

### PR DESCRIPTION
**Pull request by wallet77**
Add probes from v8 module and gc-stats module

### CODE REVIEW

File: lib/utils/module.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+var fs = require('fs');
+var debug = require('debug')('axm:profiling');
+var path = require('path');
+
+var moduleUtils = {
+  detectModule: function (moduleName, cb) {
+    require.main = require.main || {paths: ['./node_modules', '/node_modules']};
+
+    var requirePaths = require.main.paths.slice();
```
Comment (vmarchaud) [inline lib/utils/module.js]: Is `require.main.paths` still available with MJS module ?

File: lib/utils/module.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+          debug('[-] %s not found in path %s', moduleName, profilerPath);
+          requirePaths.shift();
+          return lookForModule(requirePaths);
+        });
+      } else {
+        // node version not supported
+        return cb(new Error('Node\'s version not supported'));
+      }
+    })(requirePaths);
```
Comment (vmarchaud) [inline lib/utils/module.js]: Is it mandatory for this function to self-call itself ?

File: lib/probes/v8.js [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+var utils = require('../utils/module');
+
+module.exports = function (pmx) {
+  var TIME_INTERVAL = 1000;
+
+  if (typeof v8.getHeapSpaceStatistics === 'function') {
+    var heapSpaceProbes = {
+      new_space: pmx.probe().metric({
+        name: 'New space used size',
+        type: 'v8/heap/space/new',
```
Comment (vmarchaud) [inline lib/probes/v8.js]: we should add the unit with a field `unit`

File: lib/utils/module.js (hunk 1 above)
Comment (wallet77) [reply lib/utils/module.js]: Still available on all Node version. Not tested with .mjs file.

File: lib/probes/v8.js (hunk 3 above)
Comment (wallet77) [reply lib/probes/v8.js]: Done https://github.com/keymetrics/pmx/pull/110/commits/0931cf4b8ee29393d6bb42ff23010c6dc0159d08

**Pull request merged**